identity_iota_core/src/client/iota_client.rs

```diff
@@ -95,7 +96,7 @@ impl IotaClientExt for Client {
       .map_err(|err| Error::DIDUpdateError("delete_did_output: invalid block input", Some(Box::new(err))))?
       .with_outputs(vec![basic_output])
       .map_err(|err| Error::DIDUpdateError("delete_did_output: invalid block output", Some(Box::new(err))))?
-      .with_burning_allowed(true)
+      .with_burn(Burn::new().add_alias(alias_id))
```

Comment:
@PhilippGackstatter could you please check this line? the alias output is everything we want to burn here, right?

Reply:
That's right we only want to delete the alias, so what you changed looks correct.